docs/adr/0001-provide-mechanism-to-show-hide-sections-of-accessibility-statement.md

```diff
@@ -0,0 +1,111 @@
+# Provide mechanism for showing/hiding/updating parts of the accessibility statement
+
+* Status: proposed
+* Date: 2022-02-08
+
+Technical Story: [PLATUI-1552](https://jira.tools.tax.service.gov.uk/browse/PLATUI-1552)
+
+## Context and Problem Statement
+
+Currently, some content within the existing accessibility statement template is not suitable for VOA, C-HGV, and GVMS services.
+The current implementation of the accessibility statement does not allow for hiding/showing or changing certain sections of the accessibility statement
+that some teams may require.
+
+## Decision Drivers <!-- optional -->
+
+* Services might have to find another way to host their accessibility statements
+* Contact information for VOA services is incorrect, users may not be able to contact them
+* Because information can be incorrect on the accessibility statement for VOA services, DCST might end up fielding queries they can't help with
+
+## Considered Options
+
+* Option 1: do nothing
+* Option 2: update the necessary accessibility statement fragments to cater for VOA specific context, include flags for the fragments that need to be hidden and update the accessibility statement view model to introduce these new properties.
+* Option 3: change the current accessibility statement view model and service yaml to use fragment specific properties, as currently properties are not related to specific fragments and are all related to the accessibility statement view model as a whole.
+
```

Comment:
I'm finding it a bit difficult to parse option 1 and 2, I think it might help to explode out some of the parts of the options:
* **what parts of the statement can they change?** (only bits they've asked for, or do we make it so that anything could be changed; are we going to let them replace whole section in templates, or just turn off sections and replace copy only via messages) 
* **how can they change those parts?** (via their statement yaml files, or will we have central files like "overrides/voa.yaml" that has something like "turn off X section, change copy in section Y; for services A, B, C")
* **how do we control access to making changes?** (do we allow everyone to make the changes, or do we want to limit who can, and how do we do that? If we have a separate overrides file we can use CODEOWNERS, otherwise we would have to implement an integration test)

Reply:
it feels like some of these may be difficult to consider without some examples of the API, maybe we should create some draft PRs (just to show changes teams would need to make / we might need to make, without showing how we would implement it all working.) Or a google doc with before/after code snippets

Reply:
(that said, @kyle-bowden I should have read down a bit more before I put all this comment, because I didn't yet read the code examples below, I'll look now, sorry for jumping gun before I finished the whole thing!)

Reply:
@oscarduignan thank you, I am new to writing ADR so I am sure there is plenty of room for improvement here.

Reply:
@oscarduignan I do think having small PR's with some code examples makes sense, it's always easier to understand something my example.